### PR TITLE
Refactor MetalLB with Docusaurus 

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR refactors MetalLB and adds manifest-based and Helm-based details. It also adds L2, Layer 2, and HashiCorp to Vale accept.

## Review Changes

💻 [Preview](https://deploy-preview-1517--docs-spectrocloud.netlify.app/integrations/metallb)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-440)

### CODE REVIEW

File: docs/docs-content/integrations/metallb.md [hunk 1]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
 
 MetalLB is a load-balancer implementation for bare metal [Kubernetes](https://kubernetes.io/) clusters, using standard routing protocols. This integration is recommended for the on-prem cloud(s) and will help external service(s) get an IP address when the service type is set as LoadBalancer.
 
-## MetalLB Pack Working Details:
+MetalLB deploys a controller and a speaker. The speaker is deployed as a DaemonSet that runs on all nodes.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:15]:
```suggestion
MetalLB deploys a controller and a speaker. The speaker is deployed as a DaemonSet on all nodes.
```

We can remove the able sentence and simplify the sentence.

File: docs/docs-content/integrations/metallb.md [hunk 2]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
-		kubectl describe cm config -n metallb-system.
+<Tabs queryString="versions">
 
-* However, the controller and speaker pods are already running with a previous copy of the configMap and these deployments are not aware of the new changes made to configMap. To ensure the address change are reflected, we need to restart the controller and speaker pods so that they will fetch the new configMap and start assigning new addresses correctly.
+<TabItem label="0.13.x" value="0.13.x">
 
-* Run the following commands, which will help restart the controller and speaker:
+## Prerequisites
+
+- A Kubernetes cluster running Kubernetes 1.13.0 or later that does not already have network load-balancing functionality.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:26]:
```suggestion
- A Kubernetes cluster with Kubernetes version 1.13.0 or later that does not already have network load-balancing functionality.
```

File: docs/docs-content/integrations/metallb.md [hunk 3]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+
+
+- A cluster network configuration that can co-exist with MetalLB.
+
+
+- Some IPv4 addresses for MetalLB to hand out.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:32]:
```suggestion
- Ensure sufficient IPv4 addresses for MetalLB are available.
```
The sentence is a bit too informal. @ritawatson we should ask Kevin if has a number we can recommend.

File: docs/docs-content/integrations/metallb.md [hunk 4]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+```yaml
+manifests:
+  metallb:
+    images:
+      controller: ""
+      speaker: ""
+    namespace: "metallb-system"
+    avoidBuggyIps: true
+    addresses:
+		- 192.168.250.48-192.168.250.55
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:72]:
```suggestion
    addresses:
    - 192.168.250.48-192.168.250.55
```
Fixes the indentation

File: docs/docs-content/integrations/metallb.md [hunk 5]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+
+<br />
+
+```yaml
+charts:
+	metallb-full:
+		metallb:
+			speaker:
+				frr:
+					enabled: true
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:121]:
```suggestion
charts:
  metallb-full:
    metallb:
      speaker:
        frr:
          enabled: true
```
Indentation fixes

File: docs/docs-content/integrations/metallb.md [hunk 6]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
 </TabItem>
 
 <TabItem label="0.11.x" value="0.11.x">
 
-* **0.11.0**
 
-</TabItem>
+## Prerequisites
+
+- A Kubernetes cluster running Kubernetes 1.13.0 or later that does not already have network load-balancing functionality.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:136]:
```suggestion
- A Kubernetes cluster with Kubernetes version 1.13.0 or later that does not already have network load-balancing functionality.

```

File: docs/docs-content/integrations/metallb.md [hunk 7]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
-* However, the controller and speaker pods are already running with a previous copy of the configMap and these deployments are not aware of the new changes made to configMap. To ensure the address change are reflected, we need to restart the controller and speaker pods so that they will fetch the new configMap and start assigning new addresses correctly.
+<TabItem label="0.13.x" value="0.13.x">
 
-* Run the following commands, which will help restart the controller and speaker:
+## Prerequisites
+
+- A Kubernetes cluster running Kubernetes 1.13.0 or later that does not already have network load-balancing functionality.
+
+
+- A cluster network configuration that can co-exist with MetalLB.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:29]:
```suggestion
- A cluster network configuration that does not conflict with MetalLB. For more information, refer to the official Kubernetes [Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking) documentation
```
Minor touchup with an external reference to help the reader.

File: docs/docs-content/integrations/metallb.md [hunk 8]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
 
-* **0.11.0**
 
-</TabItem>
+## Prerequisites
+
+- A Kubernetes cluster running Kubernetes 1.13.0 or later that does not already have network load-balancing functionality.
+
+
+- A cluster network configuration that can co-exist with MetalLB.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:139]:
```suggestion
- A cluster network configuration that does not conflict with MetalLB. For more information, refer to the official Kubernetes [Cluster Networking](https://kubernetes.io/docs/concepts/cluster-administration/networking) documentation.
```

File: docs/docs-content/integrations/metallb.md [hunk 9]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+
+
+- Some IPv4 addresses for MetalLB to hand out.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:142]:
```suggestion
- Ensure sufficient IPv4 addresses for MetalLB are available.
```

File: docs/docs-content/integrations/metallb.md [hunk 10]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
-## Components
 
-* MetalLB controller.
-* Speaker (runs on all nodes, deployed as DaemonSet).
+
+## Troubleshooting
+
+If controller and speaker pods are not assigning new IP addresses that you provided in the MetalLB pack, it is likely pods in existing deployments do not have the latest configMap file. 
+
+Addresses you specify in MetalLB pack values go into a configMap called `config` in the `metallb-system` namespace. The MetalLB controller and speakers use the configMap as a volume mount.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:205]:
```suggestion
IP addresses you specify in MetalLB pack values go into a configMap called `config` in the `metallb-system` namespace. The MetalLB controller and speakers use the configMap as a volume mount.
```

File: docs/docs-content/integrations/metallb.md [hunk 11]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+
+Any changed IP addresses will get updated in the configMap. You can confirm this by issuing the following command.
+
+<br />
+
+```bash
+kubectl describe cm config -n metallb-system
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:212]:
```suggestion
kubectl describe cm config --namespace metallb-system
```
Longform of the command `--namespace`

File: docs/docs-content/integrations/metallb.md [hunk 12]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+<br />
+
+To ensure updated addresses are reflected in the configMap, you need to restart the controller and speaker pods so they fetch the new configMap and start assigning new addresses correctly. Issue the commands below to restart the controller and speaker.
+
+<br />
+
+
+```bash
+	kubectl rollout restart deploy controller -n metallb-system
+	kubectl rollout restart ds speaker -n metallb-system
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:229]:
```suggestion
kubectl rollout restart deploy controller --namespace metallb-system
kubectl rollout restart ds speaker --namespace metallb-system
```
Identation and longform

File: docs/docs-content/integrations/metallb.md [hunk 13]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+
 
 ## References
 
-- [MetalLB Home](https://metallb.universe.tf)
+- [MetalLB](https://metallb.universe.tf/) 
+
 
-- [MetalLB GitHub](https://github.com/metallb/metallb)
+- [MetalLB-GitHub ](https://github.com/metallb/metallb)
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:252]:
```suggestion
- [MetalLB GitHub ](https://github.com/metallb/metallb)
```
These are two separate words, so we shouldn't don't need the dash.

File: docs/docs-content/integrations/metallb.md [hunk 14]
```diff
@@ -12,56 +12,241 @@ tags: ['packs', 'metallb', 'network']
...
+```yaml
+manifests:
+  metallb:
+    images:
+      controller: ""
+      speaker: ""
+    namespace: "metallb-system"
+    avoidBuggyIps: true
+    addresses:
+		- 192.168.250.48-192.168.250.55
```
Comment (karl-cardenas-coding) [inline docs/docs-content/integrations/metallb.md:178]:
```suggestion
    addresses:
    - 192.168.250.48-192.168.250.55
```
Indentation fix

**Review by karl-cardenas-coding — APPROVED** [review]
Minor suggestions are below. Otherwise it's 👍🏻

File: docs/docs-content/integrations/metallb.md [hunk 15]
```diff
@@ -12,56 +12,243 @@ tags: ['packs', 'metallb', 'network']
...
+```
+<br />
+
+The `charts:metallb-full:metallb` parameter section provides access to all the options of the base chart that installs MetalLB. You don’t need to change anything unless you want to install MetalLB in Free Range Routing (FRR) mode. To use FRR mode, set the option to `true`, as the example shows. 
+
+<br />
+
+```yaml
+charts:
+charts:
```
Comment (ritawatson) [inline docs/docs-content/integrations/metallb.md:117]:
```suggestion
```

**Pull request merged**